lisa/tools/qemu.py

```diff
@@ -121,13 +121,19 @@ def create_vm(
 
         if cd_rom:
             cmd += f" -cdrom {cd_rom} "
+            self.node.execute(
+                f"qemu-img resize {guest_image_path} +1G", sudo=True, shell=True
+            )
 
         # kill any existing qemu process if stop_existing_vm is True
         if stop_existing_vm:
             self.delete_vm()
 
         cmd = self._configure_qemu_command_for_cpu(cmd)
 
+        # allow qemu to use 64 GB memory
+        cmd += "-m 64000 "
```

Comment:
Is it another fix or fix the out of space issue? If the host doesn't have enough memory, will it cause errors?

Reply:
I will raise another PR for it.